Configuration/TCA/tx_ats_domain_model_application.php

```diff
@@ -387,7 +387,7 @@
                     'tablenames' => 'tx_ats_domain_model_application',
                     'table_local' => 'sys_file',
                 ]
-            ], $GLOBALS['TYPO3_CONF_VARS']['GFX']['imagefile_ext'])
+            ], $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['ats']['emSettings']['fileHandling']['allowedFileExtensions'])
```

Comment:
This is better in many ways since we now don't rely on the rather broad default list of "image" extensions anymore.